console/readdvd/dvd_udf.cpp

```diff
@@ -658,49 +668,49 @@ static int UDFFileIdentifier( uint8_t *data, uint8_t *FileCharacteristics,
  * File: Location of file the ICB is pointing to
  * return 1 on success, 0 on error;
  */
-static int UDFMapICB( udf_t *udf, struct AD ICB, uint8_t *FileType,
-                      struct Partition *partition, struct AD *File ) 
-{
+static int UDFMapICB(udf_t *udf, struct AD ICB, uint8_t *FileType,
+                     struct Partition *partition, struct AD *File) {
   uint8_t *LogBlock;
   uint32_t lbnum;
   uint16_t TagID;
   struct icbmap tmpmap;
 
   lbnum = partition->Start + ICB.Location;
   tmpmap.lbn = lbnum;
-  if(GetUDFCache(udf, MapCache, lbnum, &tmpmap)) {
+  if (GetUDFCache(udf, MapCache, lbnum, &tmpmap)) {
     *FileType = tmpmap.filetype;
     *File = tmpmap.file;
     return 1;
   }
 
-  //LogBlock = dvdalign_lbmalloc(device, 1);
-  LogBlock = (uint8_t*) malloc(2048);
-  if(!LogBlock) {
+  // LogBlock = dvdalign_lbmalloc(device, 1);
+  LogBlock = (uint8_t *)malloc(2048);
+  if (!LogBlock) {
     return 0;
   }
-    
+
   do {
-    //if( DVDReadLBUDF( device, lbnum++, 1, LogBlock, 0 ) <= 0 ) {
-    if( read ( udf->dev, LogBlock, lbnum++, 1, 0 ) ) {
+    // if( DVDReadLBUDF( device, lbnum++, 1, LogBlock, 0 ) <= 0 ) {
+    if (read(udf->dev, LogBlock, lbnum++, 1, 0)) {
```

Comment:
Codacy found an issue: [Check buffer boundaries if used in a loop including recursive loops (CWE-120, CWE-20).  ](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6072101)